integration/env/common.go

```diff
@@ -0,0 +1,32 @@
+package env
+
+import (
+	"os"
+)
+
+const (
+	// EnvVarCircleCI is the process environment variable representing the
+	// CIRCLECI env var.
+	EnvVarCircleCI = "CIRCLECI"
+	// EnvVarKeepResources is the process environment variable representing the
+	// KEEP_RESOURCES env var.
+	EnvVarKeepResources = "KEEP_RESOURCES"
+)
+
+var (
+	circleCI      string
+	keepResources string
+)
+
+func init() {
+	circleCI = os.Getenv(EnvVarCircleCI)
```

Comment:
In `aws-operator` we panic if env vars aren't set. I am not a massive fan of this, but it works since this is an `init` func.
Unsure if we want to keep doing that.

Reply:
I don't think we can panic because the env vars just might not be set. e.g. when running locally.

We now panic if the Circle SHA is an empty string as that is an error condition we can trap.

Reply:
Gotcha, cool & thanks :+1: